src/chapters/configuring.xml

```diff
@@ -841,6 +841,17 @@ guaclog: INFO: All files interpreted successfully.</computeroutput>
                                         standard port of 22 will be used.</para>
                                 </entry>
                             </row>
+                            <row>
+                                <entry><parameter>sftp-host-key</parameter></entry>
+                                <entry>
+                                    <para>The known hosts entry for the SFTP server.  This
+                                        parameter is optional, and, if not provided, no verification
+                                        of SFTP host identity will be done.  If the parameter is
+                                        provided the identity of the server will be checked
+                                        against the data.</para>
```

Comment:
We should additionally note that the format of this parameter is the same as a single entry from an OpenSSH `known_hosts` file. I see that this is stated later in the "SSH Host Verification" section, which is good, but this chapter tends to be used as a parameter reference. The format of each parameter should be clear from the parameter description, with other sections supplementing that initial clarity.

Reply:
Added x3.